package.json

```diff
@@ -23,13 +23,13 @@
         "properties": "1.2.1",
         "puppeteer-core": "^5.3.1",
         "runtime-plugin-manager-clone": "0.1.0",
-        "salesforce-alm": "49.5.0",
+        "salesforce-alm": "54.8.5",
```

Comment:
Did we try running npm install and doing packExport or some tests?

Reply:
packExport worked, will do other tests